### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by pedroct** [dckiller51/bodymiscale#4] [General]
I was unable to complete the installation. I followed the steps in the documentation, but the sensors do not appear.

**Comment by dckiller51** [comment]
> Je n'ai pas pu terminer l'installation. J'ai suivi les étapes de la documentation, mais les capteurs n'apparaissent pas.

Can you put your config.yaml and bodymiscale.yaml

**Comment by sipodd** [comment]
Maybe its because of the Foldernames, which are not intuitive in the documentation.
I placed my "bodymiscale.yaml" into "custom_components/bodymiscale/" but simply copyed the entry for the configuraiton.yaml.
This references to a folder "components/" ...that i do not have on my system.

after fixing file and folder names....everything worked like a charm :-)

**Comment by dckiller51** [comment]
The problem will no longer arise following integration via HA Configuration (Settings) -> Devices & Services -> Add -> Bodymiscale.
